components/global/Header/_subnavigation.scss

```diff
@@ -6,18 +6,17 @@ $_bg-color: functions.sass-palette(neutral02);
 .c-subnav-list {
   position: absolute;
   z-index: -5;
-  width: var(--nav-width);
   max-width: 47vw;
   max-height: calc(100vh - var(--header-height));
   overflow: auto;
```

Comment:
I'm not sure what the original purpose was, but with "auto" set, there is a flash of scrollbar when opening/closing the subnav. I tested unsetting it and it was ok, but there might be a use case I'm unaware of

Reply:
good catch.  I think you mean to have this comment on `_subsubnavigation.scss` (the new level), and you're totally right, because I'm transitioning `max-height` we get a flash of scrollbar during transition.  Luckily, for every stylistic idiosyncrasy there is [a css tricks](https://css-tricks.com/hide-scrollbars-during-an-animation/).  Never encountered this one before, but seems to be working as advertised where I've tested.